libs/sdk-core/src/persist/swap.rs

```diff
@@ -271,12 +273,7 @@ impl SqliteStorage {
             min_allowed_deposit: row.get("min_allowed_deposit")?,
             max_allowed_deposit: row.get("max_allowed_deposit")?,
             last_redeem_error: row.get("last_redeem_error")?,
-            min_msat: row.get("min_msat")?,
-            proportional: row.get("proportional")?,
-            valid_until: row.get("valid_until")?,
-            max_idle_time: row.get("max_idle_time")?,
-            max_client_to_self_delay: row.get("max_client_to_self_delay")?,
-            promise: row.get("promise")?,
+            channel_opening_fees: row.get("channel_opening_fees")?,
```

Comment:
How do we ensure backward computability here? Will deserialization it work for empty column?

Reply:
Good point, I just added a test to check that:

https://github.com/breez/breez-sdk/pull/274/commits/0cf508ecb710c3e88cd0e02e2a517fadf7a09bdf#diff-cfb44f82cbd64bb9df0ce8fc3efd492f3341a5c7b700495cf113e1ef4618a3ceR420-R441